contrib/dhalion/src/main/java/org/apache/dhalion/symptom/Symptom.java

```diff
@@ -0,0 +1,32 @@
+// Copyright 2016 Twitter. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package org.apache.dhalion.symptom;
+
+/**
+ * A {@link Symptom} identifies an anomaly or a potential health issue in a specific component or
+ * in the distributed application in general. For e.g. identification of irregular processing
+ * latency.
+ */
+public abstract class Symptom {
+  long observedAt;
+  long duration;
```

Comment:
ComponentSymptom/InstanceSymptom is  a Collection of MetricsInfo that has timestamp and duration. What is the relation between timestamp/duration in MetricsInfo and observedAt/duration here?